### DISCUSSION TITLE
Access to local network from Nextcloud-AIO?

### DISCUSSION THREAD

**Discussion by tankerktv** [nextcloud/all-in-one#3257] [Questions] [answered]
Hi! I'm just started using Nextcloud-AIO in Docker, but I can't access from NextCloud External Storage to my FTP or SMB server located in my local network.

Can anyone help me please?

**Comment by szaimen** [comment, marked as answer]
Hi, can you use the internal ip-address of the FTP or SMB server to connect to it?

**Reply by tankerktv** [reply]
I have found an internal IP address of my machine and it works)

Thanks!
